src/types/composer.ts

```diff
@@ -431,7 +432,7 @@ export type AppMethodCall<T> = Expand<Omit<T, 'args'>> & {
   )[]
 }
 
-type Txn =
+export type Txn =
```

Comment:
Is this export needed?

Reply:
a fix for a typedoc warning, this is referenced in public facing abstraction but not explicitly exported